lib/child-writer/writer-proxy.js

```diff
@@ -163,16 +163,36 @@ permissions.isElevated().then((elevated) => {
         error: error.message,
         data: error.stack
       })
-      child && child.kill()
+      child && child.kill('SIGINT')
       reject(error)
     }
 
+    process.on('SIGINT', () => {
+      if (child) {
```

Comment:
I guess we can make a utility function that we can re-use in both signal handles (basically killing the child with the same signal if there is a child, or existing)

Reply:
Done.